src/handlers.jl

```diff
@@ -219,3 +219,48 @@ function emit(handler::DefaultHandler{F, O}, rec::Record) where {F<:Formatter, O
 
     flush(handler.io)
 end
+
+mutable struct Escalator{F} <: Handler{F}
+    fmt::F
+    filters::Array{Memento.Filter}
+    levels::Dict{AbstractString, Int}
+    level::AbstractString
```

Comment:
I'm not familiar with `Memento`, but does it allow for creating a custom level? If not maybe we consider this making an enum so people can't pass in bad values to:

https://github.com/invenia/Memento.jl/pull/156/files#diff-46ed174b15e7a87714aaa0c48d1ef3a1R244

Reply:
No, you can define custom levels in memento. It'll throw a key error if the level doesn't exist

Reply:
It does allow custom levels